frontend/public/basemaps.json

```diff
@@ -0,0 +1,16 @@
+{
+  "basemaps": [
+    {
+      "name": "Map",
+      "url": "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+      "attribution": "&copy; <a href=\"http://osm.org/copyright\">OpenStreetMap</a> contributors",
+      "thumbnail": "streets.jpg"
+    },
+    {
+      "name": "Satellite",
+      "url": "https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}",
+      "attribution": "Tiles &copy; Esri &mdash; Source: Esri, DigitalGlobe, GeoEye, Earthstar Geographics, CNES/Airbus DS, USDA, USGS, AeroGRID, IGN, and the GIS User Community",
+      "thumbnail": "satellite.jpg"
+    }
+  ]
+}
```

Comment:
I put this configuration in an external JSON file because I felt it would allow us to overwrite if needed with configurations specific to different environments (DEV, TEST, PROD) if needed. This follows the same pattern of the `keycloack.json` file.